src/components/AddItem.jsx

```diff
@@ -48,7 +61,7 @@ export function AddItem({ listPath }) {
 							name="daysUntilNextPurchase"
 							type="radio"
 							id="soon"
-							required
+							defaultChecked
```

Comment:
nice usage of `defaultChecked` with an uncontrolled input

Reply:
Thank you!